TodoApi.Tests/TodoApiTests.cs

```diff
@@ -0,0 +1,117 @@
+namespace TodoApi.Tests;
+using Xunit;
+using Moq;
+using Microsoft.AspNetCore.Mvc;
+using TodoApiWithControllers.Models;
+using TodoApiWithControllers.Services;
+using TodoApiWithControllers.Controllers;
+
+public class UnitTest1
+{
+    [Fact]
+    public async void PostTodoItem_CreatesATodoItem()
+    {
+        var mockTodoItemDTO = new Mock<TodoItemDTO>();
+        var mockTodoService = new Mock<ITodoService>();
+        mockTodoService.Setup(svc => svc.CreateTodoItem(It.IsAny<TodoItemDTO>()))
+            .ReturnsAsync(GetTodos()[0]);
+
+        var controller = new TodoItemsController(mockTodoService.Object);
+
+        var result = await controller.PostTodoItem(mockTodoItemDTO.Object);

```

Comment:
You generally want to assert the results, too. 
https://fluentassertions.com/introduction

Reply:
Haha, I tried, but nothing seemed to work. I'll take a look at the link, thanks